x-pack/solutions/observability/plugins/observability_shared/public/components/add_page_attachment_to_case_modal/add_page_attachment_to_case_modal.tsx

```diff
@@ -113,13 +142,18 @@ function AddToCaseButtonContent({
           persistableStateAttachmentState: {
             ...pageAttachmentState,
             summary: comment,
+            screenContext: screenContexts || [],
```

Comment:
Why screenContext is an array? Is it so that we can include later on the screenContent from the reset tabs you mentioned yesterday? (errors, alerts etc)

Reply:
Screen context is ultimately stored as an array at the Observability AI Assistant level. Each component can call `observabilityAIAssistant.setScreenContext` (pseudocode - don't remember if that's the real method name), so when we fetch it it indeed is an array.